global.json

```diff
@@ -3,13 +3,13 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "8.0.100-rc.1.23463.5",
+    "dotnet": "8.0.100",
     "vs": {
       "version": "17.7.0"
     },
     "xcopy-msbuild": "17.7.4"
```

Comment:
Ah, wait, we need to cherry-pick 2c6ce6948c09947c930a16675222f61b5c303076 into this branch or the official build will fail!

Reply:
Done!